changelog/2025-02-04-conduit-0-13-0-release.mdx

```diff
@@ -0,0 +1,63 @@
+---
+slug: '2025-02-04-conduit-0-13-0-release'
+title: Conduit v0.13.0 release
+draft: false
+---
+
+New year, new release! We're excited to announce that Conduit [v0.13.0](https://github.com/ConduitIO/conduit/releases/tag/v0.13.0) is off the press. This one includes some features we've been eager to ship for a while now.
+
+<!--truncate-->
+
+### Key Highlights
+
+{/* Include link of blog post here when it's available*/}
```

Comment:
You might be able to use this: https://conduit.io/docs/developing/connectors/connector-specification/

Reply:
@hariso good call.

Reply:
https://github.com/ConduitIO/conduit-site/pull/229/commits/79ec570535a1d8ef6ae824d357129daf938c23f7